src/server/migrations/20200602194903_users.js

```diff
@@ -0,0 +1,15 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('users', (table) => {
+    table.increments();
+    table.string('uid').notNullable();
+    table.string('name').notNullable();
+    table.string('email').notNullable();
+    table.datetime('created_at').notNullable();
+    table.datetime('updated_at').notNullable();
+    table.datetime('deleted_at');
+  });
```

Comment:
shouldn't this be timestamp instead of datetime? 
This apply to the whole commit.

Reply:
I think the both have same effect for our App. Nevertheless, they differ in many respects but not for our App.  
Both TIMESTAMP and DATETIME columns can be automatically initialized and updated to the current date and time.and it seems that best practice would be to set all the severs in UTC timezone and use TIMESTAMP datatype for dates.